docs/_docs/contributing.md

```diff
@@ -93,7 +93,7 @@ If you want to add your plugin to the [list of plugins](https://jekyllrb.com/doc
 
 ## Code Contributions
 
-Interested in submitting a pull request? Awesome. Read on. There's a few common gotchas that we'd love to help you avoid.
+Interesting in submitting a pull request? Awesome. Read on. There's a few common gotchas that we'd love to help you avoid.
```

Comment:
Interesting?

Reply:
This is copied/generated from the CONTRIBUTING.md file.

Reply:
This was fixed in #7671